SU2_CFD/src/drivers/CDiscAdjSinglezoneDriver.cpp

```diff
@@ -539,3 +541,7 @@ void CDiscAdjSinglezoneDriver::SecondaryRecording(){
   AD::ClearAdjoints();
 
 }
+
+bool CDiscAdjSinglezoneDriver::GetTimeConvergence() const{
+  return false;
+}
```

Comment:
This could be inline in the hpp file.

Reply:
Sry forgot about this one. Changed it in the last commit :)